lib/utils/thumbnail_util.dart

```diff
@@ -42,6 +42,18 @@ Future<Uint8List?> getThumbnail(EnteFile file) async {
   }
 }
 
+Future<File?> getThumbnailFile(EnteFile file) async {
+  final thumbnail = await getThumbnail(file);
+  if (thumbnail != null) {
+    final cachedThumbnail = cachedThumbnailPath(file);
+    if (!await cachedThumbnail.exists()) {
+      await cachedThumbnail.writeAsBytes(thumbnail, flush: true);
+    }
```

Comment:
If we just need the path, it might be better to move this logic above.
Assuming we are going to rename the method to `getRemoteThumbnailFile()`
```
final cachedThumbnail = cachedThumbnailPath(file);
if (await cachedThumbnail.exists()) {
        return cachedThumbnail;
}
final thumbnail = await getThumbnail(file)
if(thumbnail != null) {
 // it might be already written to this path due // `getThumbnail(file)`
  if (!await cachedThumbnail.exists()) {
      await cachedThumbnail.writeAsBytes(thumbnail, flush: true);
    }
     return cachedThumbnail;
}
...
```